test/services/bill-runs/fetch-live-bill-runs.service.test.js

```diff
@@ -126,15 +122,14 @@ describe('Fetch Live Bill Runs service', () => {
           const results = await FetchLiveBillRunsService.go(regionId, financialYearEnding, false)
 
           expect(results).to.have.length(1)
-          expect(results[0].id).to.equal('1021c5bc-673c-48fa-98dd-733b46c84f90')
+          expect(results[0].id).to.equal(billRun.id)
```

Comment:
I think this is a necessary `evil` in the test to all this to be dynamic and not static. 

I think this is safe as it will fail if the id does not match. Worth a discussion / clarification.